src/graph/test/SchemaTest.cpp

```diff
@@ -150,16 +126,17 @@ TEST_F(SchemaTest, metaCommunication) {
         ASSERT_TRUE(verifyResult(resp, expected));
     }
     {
+        // Test the tag has been existed.
         cpp2::ExecutionResponse resp;
-        std::string query = "CREATE TAG upper(name string, EMAIL string, "
+        std::string query = "CREATE TAG person(name string, email string, "
```

Comment:
"EMAIL" is to test unreserved keyword

Reply:
Here not test if the tag has been existed，it  is to test unreserved keyword.

Reply:
fixed